website/templates/profile/notifications.mako

```diff
@@ -25,10 +25,19 @@
                         <div class="form-group">
 
                             <input type="checkbox"
-                                   data-bind="checked: subscribed"/>
-                            <label data-bind="text: list"></label>
+                                   data-bind="checked: subscribed" value="Open Science Framework General"/>
+                            <label data-bind="text: list[0]"></label>
                             <p class="text-muted" style="padding-left: 15px">Receive general notifications about the OSF every 2-3 weeks.</p>
                         </div>
+                    </form>
+                    <form>
+                        <div class="form-group">
+
+                            <input type="checkbox"
+                                   data-bind="checked: subscribed" value="Open Science Framework Help"/>
+                            <label data-bind="text: list[1]"></label>
+                            <p class="text-muted" style="padding-left: 15px">Receive up to one help notification per week.</p>
```

Comment:
Maybe use more specific language here -- what exactly is a "help notification"? How about something like "Receive helpful tips on how to make the most of the OSF, up to once per week."?